components/molecules/ContributorHoverCard/contributor-hover-card.tsx

```diff
@@ -34,14 +37,15 @@ const ContributorHoverCard = ({
   const calculatedDateFromToday = dateOfFirstPr ? calcDistanceFromToday(new Date(parseInt(dateOfFirstPr))) : "-";
   return (
     <div className="w-[364px] bg-white gap-4 p-3 rounded-lg shadow-superlative flex flex-col">
-      <div>
+      <div className="flex items-center justify-between">
         <CardProfile
           dateOfFirstPR={calculatedDateFromToday}
           githubAvatar={githubAvatar}
           githubName={githubName}
           totalPRs={totalPR}
           isRoundedAvatar={true}
         />
+        {isMaintainer && <Badge text="maintainer" />}
```

Comment:
```suggestion
        {!!isMaintainer && <Badge text="maintainer" />}
```

Reply:
I've done this already in the wrapper component